### DISCUSSION TITLE
Expected amount of data needed in hours to get good model on a new language❓

### DISCUSSION THREAD

**Discussion by ErenBalatkan** [snakers4/silero-models#31] [Q&A] [answered]
## ❓ Questions and Help

How much labeled data in terms of speech hours would you say we need to train a good model on a new language?

**Comment by snakers4** [comment, marked as answer]
For minor languages (non English) with limited dialects about 200-300h per domain, at least 4-5 domains

**Reply by matanox** [reply]
What about number of different speakers?

**Reply by snakers4** [reply]
As for speakers - I am not sure, since we did not really track it when building our models, but usually with in the wild data - data quality is the issue, not the speaker diversity
As a rough estimate I would suppose hundreds / thousands

**Comment by ErenBalatkan** [comment]
Thanks, closing
